### ISSUE TITLE
TODO Add ModuleCommand to parts that should have it / Add parts

### ISSUE THREAD

**Issue by NathanKell** [KSP-RO/RP-0#93]
Now that not just every probe grants attitude control, we need avionics parts, and we need to add ModuleCommand to rocket stages that already have guidance. Example: Atlas cores should get guidance.
For adding new parts, I added a 0.813m avionics core for use with Able (Vanguard, Thor-Able/Delta, Atlas-Able, etc), but we need multiple sizes. Maybe some 2.6m and 3.05m ones in Orbital Rocketry? What about Thor-sized rockets? @ferram4 too...

**Comment by jwvanderbeck** [comment]
I feel like I'm missing context on this.

**Comment by NathanKell** [comment]
Since the start of RP-0, we had intended that only some "probe cores" give full control, whereas some (the early "probes" like Explorer and Sputnik, or the sounding rocket core) would only allow staging and right-clicking. Instead, one would need to equip an "avionics ring" capable of controlling one's LV, and such rings would be too heavy (and draw too much electricity) to be used for satellites; the Stability node, which provides the Octo2, would be the first node that allows fully controllable satellites.
I had intended the module that did this control-limiting to be part of RemoteTech (as it was when I first wrote it, for RemoteTech 1), but thus far no one's implemented that; instead, I wrote an RT-independent module.

What also remains to be done is mass-limiting avionics rings, such that the Able ring, for example, can't control a Saturn V.
